dev/com.ibm.ws.channelfw/docs/en/channelfw.htm

```diff
@@ -63,19 +63,8 @@ <h2 id="bundle_info">Writing bundles against channelfw</h2>
 [&lt;props&gt;], where the keytypes are endpoint, channel, chain, group (of chains), and factory.</li>
 </ul>
 <h2 id="config">channelfw bundle configuration</h2>
-The channelfw bundles contains configuration for the framework itself, the WsByteBuffer
-pooling code, and the TCP channel factory.
+The channelfw bundles contains configuration for the framework itself, and the TCP channel factory.
```

Comment:
Change "contains" to "contain" ("The channelfw bundles contain ...")

Or, if there is only one channelfw bundle, change the sentence to have "The channelfw bundle contains ..." (remove "s" from "bundles").

Reply:
done